Civi/CCase/Utils.php

```diff
@@ -5,13 +5,36 @@
 
 class Utils {
 
+  /**
+   * Add a case_manager join
+   *
```

Comment:
also i believe that the complexity of this method deserves some more in-depth explanation